libevm/VMFace.h

```diff
@@ -33,6 +33,13 @@ ETH_SIMPLE_EXCEPTION_VM(OutOfGas);
 ETH_SIMPLE_EXCEPTION_VM(OutOfStack);
 ETH_SIMPLE_EXCEPTION_VM(StackUnderflow);
 
+struct VmExecResult
+{
+	VmExecResult(owning_bytes_ref&& _output, bool _revertNeeded) : output(std::move(_output)), revertNeeded(_revertNeeded) {}
+
+	owning_bytes_ref output;
+	bool revertNeeded = false;
```

Comment:
This just a comment, but the name for this flag is not right. A VM user does not care about reverts, but cares if the call succeeded or failed. `status` would be better name. Also compare with similar returned type in precompileds.